substrate/bin/utils/chain-spec-builder/src/lib.rs

```diff
@@ -158,6 +161,12 @@ pub struct CreateCmd {
 	/// The chain type.
 	#[arg(value_enum, short = 't', default_value = "live")]
 	chain_type: ChainType,
+	/// The para ID for your chain.
+	#[arg(long, value_enum, short = 'p', requires = "relay_chain")]
+	pub para_id: Option<u32>,
+	/// The relay chain you wish to connect to.
+	#[arg(long, value_enum, short = 'c', requires = "para_id")]
+	pub relay_chain: Option<String>,
 	/// The path to runtime wasm blob.
 	#[arg(long, short)]
 	runtime_wasm_path: PathBuf,
```

Comment:
omni-bencher and chain-spec-builder, both being part of the omni-node story, should have consistent arguments. This seems one clear outlier. 

I suggest using `--runtime` everywhere. @michalkucharczyk do you agree?

Reply:
> I suggest using --runtime everywhere

Makes sense. Should we just rename or deprecate `runtime_wasm_path`?

Reply:
I would do deprecation for now, emit a warning, and set it to an alias to `--runtime`.